Automattic/newspack-blocks
# PT Translation: improve Donate block translation

### All Submissions:


### Changes proposed in this Pull Request:

This PR updates the Portuguese translation for the Donate block thank you message (""Your contribution is appreciated.") from "Sua contribuição é apreciada." to "A sua contribuição é fundamental." based on publisher feedback.

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Switch your site's language to Portuguese under WP Admin > Settings.
3. Add a donate block to a page; confirm that the thank you message reads "A sua contribuição é fundamental." both in the editor and on the front-end.

### Other information:
